lib/match/git_helper.rb

```diff
@@ -1,10 +1,12 @@
 module Match
   class GitHelper
-    def self.clone(git_url)
+    def self.clone(git_url, shallow_clone)
```

Comment:
Oh sorry I didn't know about any code style guidelines. If you want to, I'm happy to update my pull request.